### PR TITLE
Miscellaneous textual corrections for IDS, Cron and Netflow

### PR DESCRIPTION
**Pull request by thelinuxfr**
Minor typo fix for netflow

### CODE REVIEW

**Comment by thelinuxfr** [comment]
Sorry for PR I mixed myself in branches

File: src/opnsense/mvc/app/views/OPNsense/Cron/index.volt [hunk 1]
```diff
@@ -121,17 +121,17 @@ POSSIBILITY OF SUCH DAMAGE.
...
+                <th data-column-id="enabled" data-width="6em" data-type="string" data-formatter="rowtoggle">{{ lang._('Enabled') }}</th>
+                <th data-column-id="minutes" data-type="string">{{ lang._('Minutes') }}</th>
+                <th data-column-id="hours" data-type="string">{{ lang._('Hours') }}</th>
+                <th data-column-id="days" data-type="string">{{ lang._('Days') }}</th>
+                <th data-column-id="months" data-type="string">{{ lang._('Months') }}</th>
+                <th data-column-id="weekdays" data-type="string">{{ lang._('Weekdays') }}</th>
+                <th data-column-id="description" data-type="string">{{ lang._('Description') }}</th>
+                <th data-column-id="command" data-type="string">{{ lang._('Command') }}</th>
+                <th data-column-id="commands" data-width="7em" data-formatter="commands" data-sortable="false">{{ lang._('Edit') }} | {{ lang._('Delete') }}</th>
+                <th data-column-id="uuid" data-type="string" data-identifier="true" data-visible="false">ID</th>
```
Comment (fabianfrz) [inline src/opnsense/mvc/app/views/OPNsense/Cron/index.volt]: {{ lang._('ID') }}

**Comment by fichtner** [comment]
This will have to wait for a few more days until we've frozen the 16.1 translations with 16.1.14. :)

**Comment by fichtner** [comment]
Applied manually, thanks!

**Pull request closed without merging**